copanier/templates/base.html

```diff
@@ -43,9 +43,11 @@ <h1 id="sitename">{{ config.SITE_NAME }}</h1>
                         <a href="{{ url_for('new_delivery') }}"><i class="icon-hotairballoon"></i>&nbsp;Nouvelle distribution</a>
                     </li>
                     {% endif %}
+                    {% if request.user and (request.user.is_staff or config.HIDE_GROUPS_LINK) %}
```

Comment:
So, we need a negation here, right?

Reply:
Indeed, the wording reverse the previous logic.
Done